src/types/quizzes.d.ts

```diff
@@ -1,11 +1,12 @@
-import { ConstructorClientOptions, NetworkParameters, UserParameters } from '.';
+import { ConstructorClientOptions, NetworkParameters, UserParameters, SortOption, ResultSources, Group, Facet, Feature } from '.';
 
 export default Quizzes;
 
 export interface QuizzesParameters {
   section?: string;
   answers?: any[];
```

Comment:
Not related to this PR but can we make this required. It's already required in the implementation